packages/gatsby/lib/bootstrap/index.js

```diff
@@ -77,6 +78,8 @@ const autoPathCreator = async (program: any) => {
 
 module.exports = async (program: any) => {
   console.log(`lib/bootstrap/index.js time since started:`, process.uptime())
+  // Fix program directory path for windows env
+  program.directory = slash(program.directory);
```

Comment:
I put this here to be commun for build/develop/etc… If I put this before, I need to make many copy/past and not forget any case. (I've tried to be the earlier possible with the less duplication as possible).

But if you prefered, I can put this line on each tasks in `cli.js`, let me know what is your choice with this considerations.